src/pages/downloads/index.tsx

```diff
@@ -12,7 +12,7 @@ const Downloads: NextPage = () => {
     <>
       <SEO
         title="Downloads"
-        description="Find downloads for our software – including Paper, Velocity, and Waterfall."
+        description="Find downloads for our software – including Paper, Velocity, Folia and Waterfall."
```

Comment:
```suggestion
        description="Find downloads for our software – including Paper, Velocity, Folia, and Waterfall."
```
imo not needed but it was there before